samples/snippets/deployment_test.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2020, Google, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from google.api_core import exceptions
+
+import pytest
+
+import create_deployment
+import delete_deployment
+import get_deployment
+import get_rollout
+import list_deployments
+import update_rollout_remove_default
+
+PROJECT_ID = "gcgs-client-lib-samples-python"
+DEPLOYMENT_ID_1 = "my-deployment"
+DEPLOYMENT_ID_2 = "my-deployment-2"
+
+
+@pytest.fixture(scope="function", autouse=True)
+def teardown():
+    clean_up_resources()
+
+    yield
+
+    print("Cleaning up resources in teardown")
+    clean_up_resources()
```

Comment:
Please refactor this fixture to create deployments unique to this test and then destroy them. Deleting all of the deployments in a project is likely to fail if there are any other tests running on the project.

Examples: [1](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/ddb0ddbd758b8e1ea9edbf0b9297c40e621887c6/translate/cloud-client/translate_v3_batch_translate_text_with_glossary_test.py#L33-L54) [2](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/ee5be6df3f8e43bd7798bc56a39b7c5f273ca90f/asset/cloud-client/quickstart_batchgetassetshistory_test.py#L36-L46)

Reply:
done. Thanks the links.